src/main/cli/settings.c

```diff
@@ -56,14 +56,15 @@
 #include "fc/rc_adjustments.h"
 #include "fc/rc_controls.h"
 
+#include "flight/alt_hold.h"
+#include "flight/autopilot.h"
 #include "flight/failsafe.h"
 #include "flight/gps_rescue.h"
 #include "flight/imu.h"
 #include "flight/mixer.h"
 #include "flight/pid.h"
 #include "flight/position.h"
-#include "flight/position_control.h"
-#include "flight/alt_hold.h"
+
```

Comment:
Unintended line break but I'll stop torturing you 😀 
```suggestion
```

Reply:
ha ha